test/functional/wallet_basic.py

```diff
@@ -428,6 +429,15 @@ def run_test(self):
         assert_equal(coinbase_tx_1["transactions"][0]["blockhash"], blocks[1])
         assert_equal(len(self.nodes[0].listsinceblock(blocks[1])["transactions"]), 0)
 
+        # Check that we can retrieve the witness nonce from the coinbase
+        coinbase_txid = self.nodes[0].getblock(block_hash)['tx'][0]
+        coinbase_tx_2 = self.nodes[1].gettransaction(txid=coinbase_txid, verbose=True)
+        assert "txinwitness" in coinbase_tx_2["decoded"]["vin"][0]
```

Comment:
not sure why this in in the wallet test. It seems blockchain or segwit related. You can put it in `./test/functional/feature_segwit.py` (or ./test/functional/rpc_blockchain.py if you prefer).

Also, this assert can be left out. The next line will throw KeyError and fail the test when txinwitness is not found.